athf/mcp/server.py

```
@@ -0,0 +1,79 @@
+    """
+    global _workspace
+    _workspace = find_workspace(workspace_path)
+    load_workspace_config(_workspace)
+
+    mcp = FastMCP(
+        name="athf",
+        instructions=(
+            "ATHF (Agentic Threat Hunting Framework) server. "
+            "Provides threat hunting operations: search hunts, check ATT&CK coverage, "
+            "find similar hunts, create new hunts, run AI-powered research, and more. "
+            "Workspace: {}".format(_workspace)
```

Ruff (0.15.6)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)